MapboxMobileEvents/MMEMetricsManager.m

```diff
@@ -242,10 +231,10 @@ - (NSDictionary *)attributes {
         attributes[MMEEventDeviceLat] = @(self.metrics.deviceLat);
         attributes[MMEEventDeviceLon] = @(self.metrics.deviceLon);
     }
-    attributes[MMEEventKeyModel] = [MMEEventsManager sharedManager].commonEventData.model;
-    attributes[MMEEventKeyOperatingSystem] = [MMEEventsManager sharedManager].commonEventData.osVersion;
-    attributes[MMEEventKeyPlatform] = [MMEEventsManager sharedManager].commonEventData.platform;
-    attributes[MMEEventKeyDevice] = [MMEEventsManager sharedManager].commonEventData.device;
+    attributes[MMEEventKeyModel] = MMEEvent.deviceModel;
```

Comment:
Shouldn't we be using `sysInfoByName:"hw.machine"` here instead of `.deviceModel` twice?